### ISSUE TITLE
additional examples in scenes.yaml

### ISSUE THREAD

**Issue by house97** [krkeegan/insteon-mqtt#4]
The scenes.yaml in the scene_sync branch has some great examples of different scene configurations.
One thing I want to better understand is how the fan speed settings would map to the kpl buttons.  Given that the fanlinc has two groups, grp 1 for the light, grp 2 for the fan, the fan speeds are mapped to values in ranges between 0x00 and 0xff (as documented in FanLinc.py).  Is the value that maps to the speed set in data_1? That's what it seems, as I look at the database of one of my devices.  If so, do we need to map data_[1,3] in the scene definition, or just data_1?

an example, cribbing off of the existing example scene 1:
>  - name: lr_fan_hi
    responders:
      - lr fan: 2
        data_1: 0xff
      - lr kpl1: 6
      - lr kpl2: 6
    controllers:
      - lr kpl1: 6
      - lr kpl2: 6
      - modem
>
Also, as it is, is this sufficient to update the modem, and thereby homeassistant, if one of the KPLs presses the button, or would the modem also need to be a responder for the scene?  As I mentioned, the above is based on an example already given, and my understanding is shaky at best.

**Comment by krkeegan** [comment]
Agree, I need a few more examples in there.

As to your first question about the fanlinc and the yaml file more generally.  

1. Anything not specified is assume to be the default value.  So if the fanlinc is specified as a responder as only "fan" then the default group is 0x01 and the default data values are D1 0xff, D2 0x00(I think) and D3 (0x01).
2. All devices support the attibributes Data_1, Data_2, and Data_3.  So no matter what, you can enter the link information at a low level.
3. If I know the "pretty name" for a device I have added that as an alternative keyword as well.  So for example for a dimmer.  To set the onlevel in the link to 70% you could say 'data_1': 0xB2 or 'on_level': 70.0.

In short, just data_1 or on_level.

As to your second question.  The modem will only see button presses if it is a responder.  That said, the import and sync functions will not handle these links.  These should be created by the pair function, I haven't looked, but the KPL pair function should create these links for all of the buttons.  If it isn't, that is something we need to fix.

**Comment by respectTheCode** [comment]
`data_3` appears to be the ramp rate? what is `data_2`?

**Comment by krkeegan** [comment]
My answer is specific to responder links on a fanlinc.  Controller links are totally different and there are minor inter-device differences.

data_1 = on_level 0xff
data_2 = ramp_rate (only on the light, so only group 0/1, the fan doesn't ramp) 0x1f = light, 0x00 = fan
data_3 = the group (0/1 for light, 2 for fan) 0x01

I did just push a correction to this to the branch, I had made a typo in the default link settings.
